job_vars/os-brick-ceph.yaml

```diff
@@ -0,0 +1,296 @@
+    # This is also used for the wrong extra_specs format issue
+#    cinder type-create blank
+
+post_stack_bash: |
+    source {{ devstack_dir.devstack }}/functions
+    TEMPEST_CONFIG={{ devstack_dir.stack }}/tempest/etc/tempest.conf
+    iniset $TEMPEST_CONFIG compute volume_device_name "sdb"
+    iniset $TEMPEST_CONFIG compute min_compute_nodes {{ win2016_compute_nodes | length }}
+    iniset $TEMPEST_CONFIG compute flavor_ref 452
+    iniset $TEMPEST_CONFIG compute flavor_ref_alt 451
+    iniset $TEMPEST_CONFIG compute-feature-enabled rdp_console true
+    iniset $TEMPEST_CONFIG compute-feature-enabled block_migrate_cinder_iscsi False
+    iniset $TEMPEST_CONFIG compute-feature-enabled block_migration_for_live_migration False
+    iniset $TEMPEST_CONFIG compute-feature-enabled live_migration False
+    iniset $TEMPEST_CONFIG compute-feature-enabled interface_attach False
+    iniset $TEMPEST_CONFIG compute-feature-enabled swap_volume False
+
+    iniset $TEMPEST_CONFIG scenario img_dir "/home/ubuntu/devstack/files/"
+    iniset $TEMPEST_CONFIG scenario img_file "{{ local_conf.image_urls | basename }}"
+    iniset $TEMPEST_CONFIG scenario img_disk_format vhdx
+    iniset $TEMPEST_CONFIG scenario dhcp_client dhclient
+
+    IMAGE_REF=`iniget $TEMPEST_CONFIG compute image_ref`
+    iniset $TEMPEST_CONFIG compute image_ref_alt $IMAGE_REF
+    iniset $TEMPEST_CONFIG validation image_ssh_user ubuntu
+    iniset $TEMPEST_CONFIG validation image_alt_ssh_user ubuntu
+
+    iniset $TEMPEST_CONFIG compute build_timeout 360
+    iniset $TEMPEST_CONFIG orchestration build_timeout 360
+    iniset $TEMPEST_CONFIG volume build_timeout 360
+    iniset $TEMPEST_CONFIG boto build_timeout 360
+
+    iniset $TEMPEST_CONFIG compute ssh_timeout 360
+    iniset $TEMPEST_CONFIG compute allow_tenant_isolation True
+#    iniset $TEMPEST_CONFIG volume-feature-enabled manage_volume False
+
+tempest_tests:
+  included-tests: |
+      tempest.scenario.test_minimum_basic
+      tempest.scenario.test_stamp_pattern
+      tempest.scenario.test_volume_boot_pattern
+
+  excluded-tests: |
+      tempest.scenario.test_stamp_pattern.TestStampPattern.test_stamp_pattern
+
+
+
+## windows
+join_ad: False
+
+win_git_prep_projects:
+  - openstack/requirements
+  - openstack/nova
+  - openstack/neutron
+  - openstack/compute-hyperv
+  - openstack/os-win
+  - openstack/os-brick
+  - openstack/networking-hyperv
+
+services:
+  - nova:
+    name: nova-compute
+    description: OpenStack nova Compute Service
+    binary: "{{ win_dir.python }}\\scripts\\nova-compute.exe"
+    config: "{{ win_dir.etc }}\\nova.conf"
+    template: nova.conf
+
+# ceph plugin installs simplejson as apt package and will conflict with other devstack automation
+upper_constraints_pinning:
+  - simplejson:
+    project: 'simplejson'
+    value: 'simplejson===3.13.2'
+
+win_python_packages:
+  - pywin32
+  - pbr
+  - pymysql
+  - ovs
+
+win_cherry_picks:
+  - project: openstack/neutron
```

Comment:
This one has already merged: https://review.opendev.org/c/openstack/neutron/+/763159